### PR TITLE
Feat/wishlist and cart

### PR DESCRIPTION
**Pull request by tonymike99**
Hey! I've added the wishlist and cart page.

https://amazing-murdock-b61413.netlify.app/

image

image

### CODE REVIEW

File: assets/css/cart.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+@import url('/assets/css/styles-reset.css');
+@import url('/assets/css/styles-common.css');
+
+/* **************************************************************************************************** */
+
+/* COMMON STYLES MODIFICATION START */
+
+.main-container {
+    grid-template-areas:
+        'header header header header header'
```
Comment (humzahasan) [inline assets/css/cart.css:12]: Suggestion : You can refactor this to 2 column layout too. The width will still be same as you want below with 2fr 1fr combo

**Review by humzahasan — APPROVED** [review]
- Well commented code
- UI looks clean
- Good code strcutre maintained

Good to Merge!

File: assets/css/cart.css (hunk 1 above)
Comment (tonymike99) [reply assets/css/cart.css:12]: > Suggestion : You can refactor this to 2 column layout too. The width will still be same as you want below with 2fr 1fr combo

Thanks Humza. Will make the change 👍

**Review by archit7448 — APPROVED** [review]
Nice UI , looks good to merge

File: templates/cart.html [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+                        </div>
+                    </div>
+                </section>
+            </main>
+
+            <footer class="footer">
+                <div class="footer-item">
+                    <small
+                        >Made with
+                        <span style="color: red"
```
[the remark this answers is not in the document]
Comment (tonymike99) [reply templates/cart.html:238]: > you should not use inline styling

Oh I forgot about it.. WIll make the changes. Thanks Archit.

**Pull request merged**